packages/extension-browser/src/manifest.json

```diff
@@ -7,6 +7,7 @@
     "icons": {
         "180": "icon.png"
     },
+    "content_security_policy": "script-src 'self' https://az416426.vo.msecnd.net/; object-src 'self'",
```

Comment:
Does `applicationinsights-js` download an external resource? Shouldn't we try to have it locally and bundle it? What happens if the domain is not reachable?

Reply:
Yes, the bundled bit is just a bootstrapper that downloads the full analytics script. The bootstrapper includes stubs for supported APIs and queues calls until the full script has loaded (so worst case scenario is we don't log analytics for a given session - nothing will break). This is the recommended approach per https://github.com/Microsoft/ApplicationInsights-JS#initializing-application-insights-js-sdk-script.

If we want to bundle the *entire* script, it unfortunately doesn't support being loaded as a module (only the bootstrapper does) and [the issue to fix that](https://github.com/Microsoft/ApplicationInsights-JS/issues/213) has been open for ~2.5 years 😞